core/http-auth-aws-crt/src/main/java/software/amazon/awssdk/http/auth/aws/crt/AwsCrtV4aHttpSigner.java

```diff
@@ -30,9 +31,9 @@ public interface AwsCrtV4aHttpSigner extends AwsV4aHttpSigner {
     /**
      * Get a default implementation of a {@link AwsCrtV4aHttpSigner}
      *
-     * @return AwsCrtV4aHttpSigner
+     * @return AwsV4aHttpSigner
      */
-    static AwsCrtV4aHttpSigner create() {
-        return null;
+    static AwsV4aHttpSigner create() {
```

Comment:
`DefaultAwsCrtV4aHttpSigner` implements `AwsV4aHttpSigner` and not `AwsCrtV4aHttpSigner`. I can make it implement the other one but I'm not sure what would be the benefit of doing so, we don't use the specific types anywhere.

Reply:
I think I'm missing some context here, why doesn't `DefaultAwsCrtV4aHttpSigner` implement `AwsCrtV4aHttpSigner`? is it because `AwsCrtV4aHttpSigner` is going to be removed?

Reply:
Yeah, `AwsCrtV4aHttpSigner` at this point should be redundant and will be removed in a follow up pull request.

Reply:
It seems there may be some confusion as we also have https://github.com/aws/aws-sdk-java-v2/pull/4352#discussion_r1306622293. But we can address all that in the follow up when we cleanup the unneeded interfaces.